binary/asn1/schema.asn.value

```diff
@@ -0,0 +1,203 @@
+value Configuration ::= {
+  harts {
+      {
+        hartid {{single 1, start 0, end 0}},
+        debug {
+          trigger {
+            {
+              index {start 0, end 3},
+              mcontrol {
+                {
+                  maskmax  4,
+                  hit TRUE,
+                  addressMatch TRUE,
+                  dataMatch TRUE,
+                  timingBefore TRUE,
+                  timingAfter TRUE,
+                  sizeAny TRUE,
+                  sizeS8 TRUE,
+                  sizeS16 TRUE,
+                  sizeS32 TRUE,
+                  sizeS64 TRUE,
+                  sizeS80 TRUE,
+                  sizeS96 TRUE,
+                  sizeS112 TRUE,
+                  sizeS128 TRUE,
+                  actionMmode TRUE,
+                  actionDmode TRUE,
+                  chain TRUE,
+                  matchEqual TRUE,
+                  matchNapot TRUE,
+                  matchGreaterEqual TRUE,
+                  matchLess TRUE,
+                  matchLowMask TRUE,
+                  matchHighMask TRUE,
+                  matchNotEqual TRUE,
+                  matchNotNapot TRUE,
+                  matchNotLowMask TRUE,
+                  matchNotHighMask TRUE,
+                  m TRUE,
+                  s TRUE,
+                  u TRUE,
+                  execute TRUE,
+                  store TRUE,
+                  load TRUE
+                }
+              }
+            },
+            {
+              index {single 4},
+              mcontrol {
+                {
+                  maskmax  4,
+                  hit FALSE,
+                  addressMatch FALSE,
+                  dataMatch FALSE,
+                  timingBefore FALSE,
+                  timingAfter FALSE,
+                  sizeAny FALSE,
+                  sizeS8 FALSE,
+                  sizeS16 FALSE,
+                  sizeS32 FALSE,
+                  sizeS64 FALSE,
+                  sizeS80 FALSE,
+                  sizeS96 FALSE,
+                  sizeS112 FALSE,
```

Comment:
In this format is it allowed to omit default (false) values? That would be really helpful for users writing descriptions.

Reply:
Hmm, I remember there is something like Default syntax, not sure how to use it and what is the advantage. I will check it.